hpat/tests_perf/test_perf_series_str.py

```diff
@@ -192,7 +192,7 @@ def __init__(self, *args, **kwargs):
 
     @classmethod
     def setUpClass(cls):
-        cls.total_data_size_bytes = 1024**2 # 1Mb
+        cls.total_data_length = 10**5 + 513
```

Comment:
What values represents this variable, items, bytes?

Reply:
The variable represents length in items.